### PR TITLE
Add requests to IANA for PSA claims

### LINKED ISSUE #44: IANA registrations
Time to move the PSA claims (those that we didn't inherit from EAT) in to the [CWT registry](https://www.iana.org/assignments/cwt/cwt.xhtml#claims-registry).

Allocations will be requested in a contiguous chunk from the negative "Specification Required" block (currently, [-65536,-257]).

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fixes #44

### CODE REVIEW

**Review by SimonFrost-Arm — APPROVED** [review]

File: cddl/example/cose.diag [hunk 1]
```diff
@@ -3,20 +3,20 @@
     / protected /   h'A10126',
     / unprotected / {},
     / paylolad /    h'AA1901097818687474703A2F2F61726D2E636F6D2F
```
Comment (adrianlshaw) [inline cddl/example/cose.diag:5]: payload?

**Comment by adrianlshaw** [comment]
Numbering looks OK 👍

File: cddl/example/cose.diag (hunk 1 above)
Comment (thomas-fossati) [inline cddl/example/cose.diag:5]:
```suggestion
    / payload /     h'AA1901097818687474703A2F2F61726D2E636F6D2F
```

**Review by adrianlshaw — APPROVED** [review]

File: cddl/test/FAIL_SoftwareComponent_Measurement_missing.diag [hunk 2]
```diff
@@ -27,10 +27,10 @@
             31415161718191a1b1c1d1e1f'
           }
         ],
-      / arm_psa_security_lifecycle / -4501: 12288 / SECURED /,
+      / arm_psa_security_lifecycle / 2395: 12288 / SECURED /,
       / nonce /            10: h'000102030405060708090a0b0c0d0e0f1011
       12131415161718191a1b1c1d1e1f',
-      / arm_psa_partition_id / -4500: -1,
+      / arm_psa_partition_id / 2394: -1,
```
Comment (SimonFrost-Arm) [inline cddl/test/FAIL_SoftwareComponent_Measurement_missing.diag:33]: Slight inconsistency here between client id (in block definition) & partition id here

**Review by SimonFrost-Arm — APPROVED** [review]
Looks good. Note slight terminology mismatch (present for a while) for client/partition ID

File: cddl/test/FAIL_SoftwareComponent_Measurement_missing.diag (hunk 2 above)
Comment (thomas-fossati) [reply cddl/test/FAIL_SoftwareComponent_Measurement_missing.diag:33]: Yeah, I can't recollect exactly how we ended up here, but we have also this: _`Note that the CDDL label used to be called arm_psa_partition_id.`_ in the Client ID subsection.  I'll raise an issue.

**Comment by thomas-fossati** [comment]
> Looks good. Note slight terminology mismatch (present for a while) for client/partition ID

raised #46 to address this

**Pull request merged**